fixtures/contest/irledflat-5x12SIXb.json

```diff
@@ -0,0 +1,614 @@
+        },
+        {
+          "dmxRange": [166, 170],
+          "type": "Effect",
+          "effectName": "Macro 3",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [171, 175],
+          "type": "Effect",
+          "effectName": "Macro 4",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [176, 180],
+          "type": "Effect",
+          "effectName": "Macro 5",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [181, 185],
+          "type": "Effect",
+          "effectName": "Macro 6",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [186, 190],
+          "type": "Effect",
+          "effectName": "Macro 7",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [191, 195],
+          "type": "Effect",
+          "effectName": "Macro 8",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [196, 200],
+          "type": "Effect",
+          "effectName": "Macro 9",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [201, 205],
+          "type": "Effect",
+          "effectName": "Macro 10",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [206, 210],
+          "type": "Effect",
+          "effectName": "Macro 11",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [211, 215],
+          "type": "Effect",
+          "effectName": "Macro 12",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [216, 220],
+          "type": "Effect",
+          "effectName": "Macro 13",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [221, 225],
+          "type": "Effect",
+          "effectName": "Macro 14",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [226, 233],
+          "type": "Effect",
+          "effectName": "Music-sensitive Macro 1 (colour sequencing)",
```

Comment:
Please use American English here: *Color*. Or is it a `ColorJump` effect anyway?